### PR TITLE
Added support for Cardboard V2.0 style touch-triggers

### PR DESCRIPTION
**Pull request by sean-purcell**
Also deprecated - (void)magneticTriggerPressed and added - (void)triggerPressed to be more generic.

### CODE REVIEW

**Comment by rsanchezsaez** [comment]
Hey @iburinoc, thanks a lot for your contribution, this is a worthwhile change.

Some comments:
- Replacing `-(void)magneticTriggerPressed` by `-(void)triggerPressed` is fine. But I'd rather directly rename the methods instead of deprecating the old one. The change is very minor, so I think in this instance it's ok to break previous code.
- I'd directly add the touch-as-trigger functionality to `CBDViewController.mm/h`, since this part is iOS specific rather than more generic as the rest of the C++ SDK. Also, I'd add a (on by default if you want) `useTouchTrigger` property to `CBDViewController` to optionally disable this feature.

If you do these changes I'll merge the pull request. :)

**Comment by sean-purcell** [comment]
That sounds reasonable, I implemented the changes :)

File: CardboardSDK/CBDViewController.mm [hunk 1]
```diff
@@ -271,14 +273,23 @@ - (void)setNeckModelEnabled:(BOOL)neckModelEnabled
...
+    if ([self.stereoRendererDelegate respondsToSelector:@selector(triggerPressed)])
     {
-        [self.stereoRendererDelegate magneticTriggerPressed];
+        [self.stereoRendererDelegate triggerPressed];
     }
 }
 
+- (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
+{
+    if(self.useTouchTrigger) {
```
Comment (rsanchezsaez) [inline CardboardSDK/CBDViewController.mm]: Sorry for the nit, could you add a space between the `if` and the parens? :)

Comment (sean-purcell) [inline CardboardSDK/CBDViewController.mm]: Done, sorry for missing the style there

**Comment by rsanchezsaez** [comment]
Done, thanks a lot for your contribution!

**Pull request merged**